### ISSUE TITLE
i18n

### ISSUE THREAD

**Issue by mmaelicke** [sumo-rhine/kinamo#15]
@JesJehle , @ArcticPuffin , are there any plans to publish kinamo in other languages than english?

If yes, that would be an urgent discussion point. i18n is build DEEPLY into js frontends. 
Just to give you an idea: 

https://react.i18next.com/guides/quick-start

**Comment by JesJehle** [comment]
Yes, we need translates for German and France, but I thought we create all translations manually and provide them based on the IP address of the user.

**Comment by mmaelicke** [comment]
At the end of the day, that is how the different i18n implementations handle it in all js frameworks. (well, not IP, but browser language, but fallback to IP if that is not set).
The translations still have to be created manually, but you need a maintainable way of how to inject the translation. Copying the whole project is a bad idea as it will turn into a gigantic maintenance nightmare rather quickly...

So, I will have a look into i18n in React to see where we have to be careful...
